src/store/modules/bot-commands/bot-commands-mutations.ts

```diff
@@ -1,14 +1,27 @@
 import { MutationTree } from 'vuex'
 import { BotCommand, BotCommandsState } from '@/store/modules/bot-commands/types.ts'
+import { uniq } from 'lodash'
 
 export const mutations: MutationTree<BotCommandsState> = {
   addCommand(state, botCommand: BotCommand): void {
     const commandValue = botCommand.command.trim()
-    const botCommands = state.commands[botCommand.partnerId]
+    let botCommands = state.commands[botCommand.partnerId]
     if (!botCommands) {
-      state.commands[botCommand.partnerId] = [commandValue]
-    } else if (!botCommands.includes(commandValue)) {
+      botCommands = [commandValue]
+    } else {
+      const index = botCommands.indexOf(commandValue)
+      if (index >= 0) {
+        botCommands.splice(index, 1)
+      }
       botCommands.push(commandValue)
     }
+    state.commands[botCommand.partnerId] = botCommands
+  },
+
+  initCommands(state, { partnerId, commands }: { partnerId: string; commands: string[] }): void {
+    const botCommands = state.commands[partnerId]
+    if (!botCommands) {
+      state.commands[partnerId] = uniq(commands)
```

Comment:
Doesn't seem that `uniq()` works as we need:

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/97a5108d-9046-4c07-86ed-f8be55b6438d)

**Expected result:**
...
/orders ETH/USDT
/params
/start mm
/orders
/stop mm